akka-sample-distributed-data-java/src/multi-jvm/scala/sample/distributeddata/ShoppingCartSpec.scala

```diff
@@ -29,18 +35,19 @@ class ShoppingCartSpecMultiJvmNode1 extends ShoppingCartSpec
 class ShoppingCartSpecMultiJvmNode2 extends ShoppingCartSpec
 class ShoppingCartSpecMultiJvmNode3 extends ShoppingCartSpec
 
-class ShoppingCartSpec extends MultiNodeSpec(ShoppingCartSpec) with STMultiNodeSpec with ImplicitSender {
+class ShoppingCartSpec extends MultiNodeSpec(ShoppingCartSpec) with STMultiNodeSpec {
```

Comment:
I don't see the duplicate "started with address" logging as you did. It's only the slf4j logging for me, which can be seen from the date format "2019-11-18 14:43:44,213"

Reply:
did you clean before running the test?

Reply:
It's all good after a clean.  I guess more sbt-multi-jvm bugs?  Thanks for the review.

Reply:
probably same as before, ending up with a mix of jar files in that multi-jvm run directory

Reply:
Oh yeah, I can't remember cleaning after I bump... old assumptions...